akka-actor-tests/src/test/scala/akka/actor/DeathWatchSpec.scala

```diff
@@ -83,7 +83,7 @@ trait DeathWatchSpec { this: AkkaSpec with ImplicitSender with DefaultTimeout =>
 
   "The Death Watch" must {
     def expectTerminationOf(actorRef: ActorRef) =
-      expectMsgPF(5 seconds, actorRef + ": Stopped or Already terminated when linking") {
+      expectMsgPF(5 seconds, "" + actorRef + ": Stopped or Already terminated when linking") {
```

Comment:
nitpicking: I find it unfortunate from aesthetics point of view that we need now to prepend a "". Maybe better to use string interpolation.

```suggestion
      expectMsgPF(5 seconds, s"$actorRef: Stopped or Already terminated when linking") {
```

Not a requirement for this PR. We can add it later or let it like that. Is more me sharing a thought.

Reply:
I also think that and have tried to implement [Any2StringAdd](https://github.com/ohze/scalafix-rules/blob/master/scalafix/rules/src/main/scala/fix/Any2StringAdd.scala) rule using string interpolation but I found its unnecessary complicated. So I switched to the simple solution by prefixing an empty string.